docs/guide/date-and-time-handling.md

```diff
@@ -1,17 +1,10 @@
 # Date and time handling
 
-Date and time formats can be set in the
-[configuration options](configuration-options.md).
+The formats for the default date and time parsing functions can be set using configuration options:
+- [dateFormats](../api/interfaces/configparams.md#dateformats),
+- [timeFormats](../api/interfaces/configparams.md#timeformats),
+- [nullYear](../api/interfaces/configparams.md#nullyear).
 
-`dateFormats` is a list of formats supported by the parser
-inside HyperFormula. The default format is
-`['MM/DD/YYYY', 'MM/DD/YY']`. The separator is ignored and it can
-be any of the following:  '-' (dash), ' ' (empty space),
-'/' (slash).
-
-Similar to `dateFormats`, `timeFormats` is a list of time formats
-supported by the parser. The default format is
-`['hh:mm', 'hh:mm:ss.sss']`. The only accepted separator is ':' (colon).
```

Comment:
It is described in the API ref for these configuration options. No need for duplication.